src/modules/workspace/controllers/feature.controller.ts

```diff
@@ -84,6 +84,7 @@ export class FeatureController {
   })
   @ApiResponse({ status: 200, description: "Feature Updated Successfully" })
   @ApiResponse({ status: 400, description: "Update Feature Failed" })
+  @UseGuards(JwtAuthGuard)
   async updateFeature(
     @Param("name") name: string,
     @Body() updateFeatureDto: UpdateFeatureDto,
```

Comment:
additional validation will be required here, otherwise any random user with jwt will able to modify any feature. I think its for admin use only, for now can create save a custom dummy key in env,  use this as a check, if anyone required to update feature will pass the key in param or body.

Reply:
This feature is still a experimental feature, we will make this API private in future.

Reply:
Created a different issue to track making the API endpoint private. We can close this.